mods/farming/init.lua

```diff
@@ -66,7 +78,16 @@ minetest.register_craft({
 	}
 })
 
+minetest.register_craft({
+	output = "farming:string 3",
+	recipe = {
+		{"farming:cotton", "farming:cotton", "farming:cotton"},
+	}
+})
```

Comment:
Imho a vertical row would be better. 3 items of the same type in a horizontal row reminds to a slab recipe. But eg. technic also has a recipe to make a wire item vertical.